galactic/distribution.yaml

```diff
@@ -2818,13 +2818,10 @@ repositories:
       url: https://gitlab.com/ApexAI/performance_test.git
       version: 1.0.0
     release:
-      packages:
-      - performance_report
```

Comment:
@revati-naik same question as in https://github.com/ros/rosdistro/pull/33763#discussion_r912195974

Reply:
The package is being removed since it has dependencies that cannot be met with build farm releases. See https://github.com/ros/rosdistro/pull/33341